### DISCUSSION TITLE
playing SACD iso files on the fly

### DISCUSSION THREAD

**Discussion by GuannanHu** [Taiko2k/Tauon#1001] [Ideas]
update: There is already tools in python to achieve this. Indeed as Taiko2k mentioned,  probably too bloated if  incorporate them into Tauon. 
>  I want to add the SACD features to Tauon, such as playing SACD iso files, converting them to FLAC, etc.

**Comment by Taiko2k** [comment]
I don't think its worth supporting super rare formats like that one. Its not convenient. Better to just use an external tool to convert to FLAC for playback and keep your ISO's archived.
